plugins/search/src/components/SearchPage/SearchPage.test.tsx

```diff
@@ -106,9 +140,27 @@ describe('SearchPage', () => {
       '?query=bieber&types[]=software-catalog&pageCursor=SOMEPAGE&filters[anyKey]=anyValue',
     );
 
-    await renderInTestApp(<SearchPage />);
+    await renderSearchPageWithConfig();
 
     const calls = (window.history.replaceState as jest.Mock).mock.calls[0];
     expect(calls[2]).toContain(expectedLocation);
   });
+
+  it('initializes the SearchPage with a custom page limit', async () => {
+    const { getByText } = await renderSearchPageWithConfig({
```

Comment:
Nice ⭐

Reply:
i'll gently hint at `screen`, being the superior way of getting parts of the rendered output :) Example: https://github.com/backstage/backstage/blob/cc14e1ba0ec638dbc9c07636e8db89e3cfd2e5cc/packages/core-components/src/components/SupportButton/SupportButton.test.tsx#L56